guides/accessibility.md

```diff
@@ -22,6 +22,10 @@ POUR principles and get a checklist for these guidelines,
 
 Testing for accessibility should be done throughout the project lifecycle.
 
+### Accessibility audits
+
+Visit the CFPB's internal [Accessibility documentation [GHE]](https://GHE/CFPB/hubcap/wiki/Accessibility) to learn how to audit your project work in a standardized manner.
```

Comment:
Because this isn't a valid link it'll won't go anywhere, would probably be better to leave the brackets off the running text and leave the link in the parens, so everyone can copy/paste then edit the GHE bit.

Reply:
I'll split the difference and do both - i do tend to use the links and then just replace the address in the URL bar, guess I'm quirky like that